hack/istio/tempo/install-tempo-env.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+##############################################################################
+# install-tempo-env
+#
+# Installs the Tempo environment using the Tempo operator.
+#
+# See --help for more details on options to this script.
+#
+##############################################################################
+
+CLIENT_EXE="kubectl"
+DELETE_ALL="false"
+DELETE_TEMPO="false"
+INSTALL_BOOKINFO="true"
+INSTALL_ISTIO="true"
+INSTALL_KIALI="true"
```

Comment:
there is no option to set this (no `-ik|--install-kiali`)

Reply:
Right, added!